test/deploy_monitoring_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestDeployMonitoring(t *testing.T) {
+	t.Parallel()
+
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "./unit-test",
+	})
+
+	defer terraform.Destroy(t, terraformOptions)
+
+	terraform.InitAndApply(t, terraformOptions)
+
+	// options := k8s.NewKubectlOptions("", "", "opa")
+
+	// service := k8s.GetService(t, options, "opa")
+	// require.Equal(t, service.Name, "opa")
```

Comment:
Any chance you can either define what these comments are, or just remove them?

Reply:
Fixed, thank you